docs/user-guide/concepts.md

```diff
@@ -87,6 +88,8 @@ namespaces:
   as isolation units for their own services. However, namespace creation is a
   privileged cluster-level operation, and you typically want to control this
   privilege very closely.
+* You might want to give some amount of resources to a team (similar to ResourceQuota), and they can
```

Comment:
ResourceQuota in backquotes, e.g. `ResourceQuota`

Also below for `HierarchicalResourceQuota`

Reply:
Can you add the backquotes? E.g. instead of:

```
(similar to ResourceQuota)
```

it should look like this:

```
(similar to `ResourceQuota`)
```

And ditto everywhere else in this change where you say either ResourceQuota or HierarchicalResourceQuota.

Reply:
well i guess i will change it for not only my text then